mirrord/cli/src/port_forward.rs

```diff
@@ -268,6 +288,45 @@ impl PortForwarder {
                     );
                 }
             },
+            DaemonMessage::GetAddrInfoResponse(GetAddrInfoResponse(message)) => match message {
```

Comment:
Not quite sure what's going on in this branch.
1. The record can contain more than 1 entry
2. `else` block inside `Ok(DnsLookup(record))` branch looks different than `Err(error)` branch, but it's practically the same case, I think

I'd do sth like
```rust
DaemonMessage::GetAddrInfoResponse(GetAddrInfoResponse(message)) => match message {
    Ok(DnsLookup(record)) if !record.is_empty() => {
        // yay
    }
    error_or_empty => {
        // nay
    }
}
```